### DISCUSSION TITLE
umi开启icons配置，使用本地svg文件作为icon使用

### DISCUSSION THREAD

**Discussion by liujiajun1994** [umijs/umi#10919] [Q&A]
版本：
![image](https://user-images.githubusercontent.com/44549884/230536833-4cd625b4-5705-4067-9df0-fdfd8658a356.png)
报错：
![image](https://user-images.githubusercontent.com/44549884/230536542-0774a78b-494d-4fdf-87bd-a2f443a1d838.png)

**Comment by fz6m** [comment]
给一个最小复现看看，日志不清楚。
